Dockerfile

```diff
@@ -1,17 +1,12 @@
-FROM golang:alpine as build
-LABEL org.opencontainers.image.source = "https://github.com/containerish/OpenRegistry"
-
+FROM golang:1.17 as build
```

Comment:
@pratikbin while we're here, let's bump it to `v1.19`? Here and in `go.mod`. We don't use any `Go 1.17` specific features so we should be good here. Also let's keep alpine here. Reason being alpine is almost 3 times lighter than this image. More on why in the comment below